DQM/HcalTasks/plugins/QIE10Task.cc

```diff
@@ -219,128 +179,52 @@ QIE10Task::QIE10Task(edm::ParameterSet const& ps):
 		HcalDetId did = frame.detid();
 		HcalElectronicsId eid = HcalElectronicsId(_ehashmap.lookup(did));
 
+		int fakecrate=-1;
+		if (eid.crateId() == 22) fakecrate = 0;
+		if (eid.crateId() == 29) fakecrate = 1;
+		if (eid.crateId() == 32) fakecrate = 2;
+		int index = fakecrate*12+eid.slot()-1;
```

Comment:
No, this would be bad. I added a protection, skipping the digi if the crate number is not in [22,29,32].